docs/can-observe.md

```diff
@@ -25,33 +26,37 @@ dog.name = 'Wilbur';
 
 ## Use
 
-Using __can-observe__ allows you to create observable objects for which any property added can be observed. It is like [can-map] but without the required [can-map.prototype.attr attr] method. This makes can-observe ideal for use-cases where the data that needs to be observed might be dynamic, or where the more rigid approach of [can-define] is unneeded.
+Using `can-observe` allows you to create observable objects with any property added can be observed, including nested objects. It is like [can-map] but without the required [can-map.prototype.attr attr] method. This makes `can-observe` ideal for use-cases where the data may be dynamic, or where the more rigid approach of [can-define] is unneeded.
 
-To use can-observe call the observe method with an object for which to create a proxy for. This will return a [Proxy](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy) object for which any changes will then reflect back on the target object.
+To use `can-observe` call the `observe()` method with an object. This will return a [Proxy](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy) object where any changes will reflect back on the target object. Nested objects will be observed lazily when they are accessed or set dynamically after initialization on a `can-observe` proxy object.
+
+To listen for changes on a property, use [can-reflect/observe.onKeyValue canReflect.onKeyValue]. Pass the handler as the third argument which will be triggered when a new value is set on the proxy. In the example below, updating `dog.name` to `"Wilbur"` will trigger the callback that will `console.log` the new value `"Wilbur"`.
```

Comment:
I'm not sure showing how to listen to changes it the "right" first example.  It almost should be using it with `can-stache`.  No one should really be listening to observe directly at first.